mail_activity_board/views/mail_activity_view.xml

```diff
@@ -0,0 +1,216 @@
+                                    <strong>
+                                        <t t-esc="activity_type_id" />
+                                    </strong>
+                                </div>
+                            </div>
+                            <strong><span t-attf-class="o_activity_date o_activity_color_#{record.state}"><t t-esc="record.label_delay" /></span></strong>
+                            <div>
+                                <strong class="o_kanban_record_subtitle"><field name="res_name"/></strong>
+                            </div>
+                            <div>
+                                <span class="o_kanban_record_subtitle"><field name="user_id"/></span>
+                            </div>
+                            <field name="calendar_event_id_start"/>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+         </field>
+    </record>
+
+
+    <!-- SEARCH VIEW -->
+    <record id="mail_activity_view_search" model="ir.ui.view">
+        <field name="name">search.activity.view.calendar.inherit.activities_board</field>
+        <field name="model">mail.activity</field>
+        <field name="inherit_id" ref="mail.mail_activity_view_search"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr='//field[@name="res_model_id"]' position='before'>
+                <field name="user_id"/>
+                <field name="res_name" string="Origin (opportunity/customer)"/>
+            </xpath>
+
+            <xpath expr='//filter[@name="activities_my"]' position='after'>
+                <filter string="Act. next month" name="activities_month"
+                        domain="[('date_deadline', '&lt;', (context_today()+datetime.timedelta(days=30)).strftime('%Y-%m-%d'))]"
+                        help="Show activities which date deadline is before than 1 month."/>
+                <filter string="Act. next 6 months" name="activities_6_month"
+                        domain="[('date_deadline', '&lt;', (context_today()+datetime.timedelta(days=180)).strftime('%Y-%m-%d'))]"
+                        help="Show activities which date deadline is before than 6 months."/>
+                <separator/>
+            </xpath>
+
+            <xpath expr='//search/group' position='inside'>
+                <filter string="User" domain="[]" groups="sales_team.group_sale_salesman_all_leads" context="{'group_by':'user_id'}"/>
+                <filter string="Origin" context="{'group_by': 'res_model_id'}"/>
+            </xpath>
+
+        </field>
+    </record>
+
+
+<!--
+    ACTION
+-->
+
+    <record model="ir.actions.act_window" id="open_boards_activities_form_tree">
+        <field name="name">ACTIVITIES</field>
+        <field name="res_model">mail.activity</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
```

Comment:
I think it's better to default to the kanban view.

Reply:
I've already modified it.